packages/react/src/components/popover/Popover.tsx

```diff
@@ -23,13 +24,20 @@ export interface ITetherComponentCopiedProps {
     onRepositioned?: (...args: any[]) => void;
 }
 
+interface PopoverOwnProps {
+    /**
+     * Whether the popover has dropdowns components rendered in it. The goal of this prop is to prevent the popover from closing when selecting a dropdown value.
+     */
+    hasDropdowns?: boolean;
```

Comment:
Do we need this props? I feel like the new behaviour would be less confusing and adding a prop for a specific case goes against our recent discussions

Reply:
My concern was that `Defaults.DROP_ROOT = 'body'` by default so for someone that didn't change it, the popover would never close

Reply:
Makes sense to me! as long as it is clear